### ISSUE TITLE
BytesTransmitted and BytesReceived should be long type in LinuxIPInterfaceStatistics

### ISSUE THREAD

**Issue by JianpingLiu0225** [dotnet/corefx#34139]
file `corefx/src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxIPInterfaceStatistics.cs`
you have

```c#
        public LinuxIPInterfaceStatistics(string name)
        {
            _table = StringParsingHelpers.ParseInterfaceStatisticsTableFromFile(NetworkFiles.InterfaceListingFile, name);

            // sys/class/net/<interfacename>/tx_queue_len
            string transmitQueueLengthFilePath = Path.Combine(NetworkFiles.SysClassNetFolder, name, NetworkFiles.TransmitQueueLengthFileName);
            _transmitQueueLength = StringParsingHelpers.ParseRawIntFile(transmitQueueLengthFilePath);

        }
        public override long BytesReceived { get { return _table.BytesReceived; } }
        public override long BytesSent { get { return _table.BytesTransmitted; } }
```

where `_table.BytesReceived` and `_table.BytesTransmitted` returned in `StringParsingHelpers.ParseInterfaceStatisticsTableFromFile` are uint type which has a max value of 2^32 - 1. 

the fix needs to be in the `corefx/src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Statistics.cs` to use `long` type in the struct `IPInterfaceStatisticsTable`

**Comment by danmosemsft** [comment]
@VasiliyNovikov has https://github.com/dotnet/corefx/pull/34119 up. Is that sufficient?

**Comment by davidsh** [comment]
Fixed with PR #34119
